src/uagents/config.py

```diff
@@ -27,6 +27,7 @@ class AgentNetwork(Enum):
 AGENT_NETWORK = AgentNetwork.FETCHAI_TESTNET
 
 AGENTVERSE_URL = "https://agentverse.ai"
+ALMANAC_API_URL = "https://staging.agentverse.ai/v1/almanac/"
```

Comment:
Can we just use `AGENTVERSE_URL + "/v1/almanac"` instead?

The intention was that if people want to use the staging environment, they can create their agents with:
```python
agent = Agent(agentverse="https://staging.agentverse.ai")
```

Reply:
Makes sense!

Is the almanac API already reachable on the production environment, or do we need to relay all agents to staging for the time being?

Reply:
Yes, it should be fine on production.